deploy/resources/addons/04-consolecli/console_cli_download_tkn.yaml

```diff
@@ -12,3 +12,7 @@ spec:
       text: Download tkn for Mac
     - href: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-windows-amd64-0.13.1.zip
       text: Download tkn for Windows
+    - href: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/tkn-linux-ppc64le-0.13.1.tar.gz
+      text: Download tkn for Power
```

Comment:
Can you change this to IBM Power and similarly below one to IBM Z

Reply:
Done :+1: